src/taxman.py

```diff
@@ -77,13 +77,18 @@ def _evaluate_taxation_GERMANY(self, coin: str, operations: list[Operation]) ->
             elif isinstance(op, Buy):
                 bilance.put(op)
             elif isinstance(op, Sell):
-                try:
-                    sold_coins = bilance.sell(op.change)
-                except IndexError:
-                    if coin == config.FIAT:
+                sold_coins = bilance.sell(op.change)
+                if sold_coins is None:          # Queue ran out of items to sell...
```

Comment:
I'd like to stick to autopep8 for formatting (using 2 spaces before the inline comment). Otherwhise my autoformatter will change it after my next commit :D

Depending on your IDE you could use to one. I would recommend using an autoformatter. Otherwhise ignore this comment.

We may even want to avoid inline comments and move them in their resepctive if/else-clause.

Reply:
Thanks for the info, I'll stick to that!